src/ts2lang-app.ts

```diff
@@ -18,4 +18,6 @@ export function run() {
         .parse(process.argv);
 
     main(program["file"]);
-}
\ No newline at end of file
+}
+
+if (!module.parent) { run(); }
```

Comment:
This change was made to make the vscode laucher (F5) work (for debug purposes) - if we change to the main file (generated) it won't be necessary 👍